api/src/Serializer/Normalizer/TranslationConstraintViolationListNormalizer.php

```diff
@@ -50,7 +50,10 @@ public function normalize(mixed $object, string $format = null, array $context =
 
                     $translations = $this->translateToAllLocalesService->translate(
                         $violation->getMessageTemplate(),
-                        $violation->getParameters()
+                        array_merge(
+                            $violation->getPlural() ? ['%count%' => $violation->getPlural()] : [],
```

Comment:
I did not see directly in which case you needed this.
A test which covers this path would be nice.

Reply:
Test added in https://github.com/ecamp/ecamp3/pull/3465/commits/f4baba30b060b94806c8b7803f4368fd5010fc30

Many symfony internal validators have violations messages that have a singular and a plural version. The `%count%` parameter needs to be provided to the translator such that it selects the right version:
https://github.com/symfony/symfony/blob/6.3/src/Symfony/Contracts/Translation/TranslatorInterface.php#L22
 https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/Validator/Violation/ConstraintViolationBuilder.php#L123

Without this adjustment, the output would be the following (both singular+plural version returned):
```php
'translations' => [
    'en' => 'This value is too long. It should have 32 character or less.|This value is too long. It should have 32 characters or less.',
    'de' => 'Diese Zeichenkette ist zu lang. Sie sollte höchstens 32 Zeichen haben.|Diese Zeichenkette ist zu lang. Sie sollte höchstens 32 Zeichen haben.',
    'fr' => 'Cette chaîne est trop longue. Elle doit avoir au maximum 32 caractère.|Cette chaîne est trop longue. Elle doit avoir au maximum 32 caractères.',
    'it' => 'Questo valore è troppo lungo. Dovrebbe essere al massimo di 32 carattere.|Questo valore è troppo lungo. Dovrebbe essere al massimo di 32 caratteri.',
],
```